cmd/sourced/compose/workdir/org.go

```diff
@@ -0,0 +1,26 @@
+package workdir
+
+import (
+	"encoding/base64"
+	"strings"
+)
+
+// InitWithOrgs initialize workdir with remote list of organizations
+func InitWithOrgs(orgs []string, token string) (string, error) {
+	workdir := base64.StdEncoding.EncodeToString([]byte(strings.Join(orgs, ",")))
+	workdirPath, err := absolutePath(workdir)
```

Comment:
We mentioned that workdirs from orgs will live in `orgs` dir. It can be done in another PR, but it could be added in `TODO` PR description to avoid forgetting it

Reply:
thanks. updated

Reply:
Is it possible you forgot to push? I don't see this change

Reply:
TODO is updated. Not the code.

Reply:
oh, you mean updated the description, I missed it on the first read

> move orgs workdirs into orgs subdir and local workdirs into local subdir (maybe)